### PR TITLE
[New Presence] Glimesh Presence v1.0.0

### PR DESCRIPTION
**Pull request by ItzNxthaniel**
New Presence: For [Glimesh.tv](https://glimesh.tv/)
Tested for any bugs and such, and should all be fixed! :)

### CODE REVIEW

File: websites/G/Glimesh/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    "contributors": [
+        {
+            "name": "zjtech123",
+            "id":  "261236127581601793"
+        }
+    ],
+    "category": "socials",
+    "logo": "https://cdn.discordapp.com/attachments/406924917557297173/825642515486867486/logo-with-text-sm.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/406924917557297173/825642542128300072/unknown.png",
```
Comment (Bas950) [inline websites/G/Glimesh/dist/metadata.json:16]: upload to imgur

File: websites/G/Glimesh/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      "name": "Nathaniel.",
+      "id": "147891648628654082"
+    },
+    "contributors": [
+        {
+            "name": "zjtech123",
+            "id":  "261236127581601793"
+        }
+    ],
+    "category": "socials",
```
Comment (Bas950) [inline websites/G/Glimesh/dist/metadata.json:14]: shouldnt it be under the Video and Streams category?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/Glimesh/dist/metadata.json (hunk 2 above)
Comment (ItzNxthaniel) [reply websites/G/Glimesh/dist/metadata.json]: Newest commit has changes made! 👍

File: websites/G/Glimesh/dist/metadata.json (hunk 1 above)
Comment (ItzNxthaniel) [reply websites/G/Glimesh/dist/metadata.json]: Newest commit has changes made! 👍

File: websites/G/Glimesh/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    "author": {
+      "name": "Nathaniel.",
+      "id": "147891648628654082"
+    },
+    "contributors": [
+        {
+            "name": "zjtech123",
+            "id":  "261236127581601793"
+        }
+    ],
```
Comment (ririxidev) [inline websites/G/Glimesh/dist/metadata.json:13]: image

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @ItzNxthaniel!
You should check out thing(s) that I requested to change in my review. Please look at them!
- Also please remove these comments in code, they are not necessary.

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

**Comment by ItzNxthaniel** [comment]
> Heya @ItzNxthaniel!
> You should check out thing(s) that I requested to change in my review. Please look at them!
> - Also please remove these comments in code, they are not necessary.
> 
> Thanks,
> **ririxi**. PreMiD Reviewer Team.
> Contact: mail@ririxi.dev, ririxi#2721

Sorry about that I added him because he made the assets that I used. I'll remove him asap. Also is removing the comments necessary? I'm trying to get better on commenting my code. 😂🥴

File: websites/G/Glimesh/dist/metadata.json (hunk 3 above)
Comment (ItzNxthaniel) [reply websites/G/Glimesh/dist/metadata.json]: Requested Changes, fixed in newest commit! :)

**Review by ririxidev — APPROVED** [review]
Lgtm

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**